crates/ruff_linter/src/rules/pygrep_hooks/rules/blanket_noqa.rs

```diff
@@ -55,35 +55,23 @@ impl Violation for BlanketNOQA {
 
     #[derive_message_formats]
     fn message(&self) -> String {
-        let BlanketNOQA {
-            missing_colon,
-            space_before_colon,
-            file_exemption,
-        } = self;
-
         // This awkward branching is necessary to ensure that the generic message is picked up by
         // `derive_message_formats`.
-        if !missing_colon && !space_before_colon && !file_exemption {
-            format!("Use specific rule codes when using `noqa`")
-        } else if *file_exemption {
-            format!("Use specific rule codes when using `ruff: noqa`")
-        } else if *missing_colon {
-            format!("Use a colon when specifying `noqa` rule codes")
+        if !self.missing_colon && !self.space_before_colon && !self.file_exemption {
+            "Use specific rule codes when using `noqa`".to_string()
+        } else if self.file_exemption {
+            "Use specific rule codes when using `ruff: noqa`".to_string()
+        } else if self.missing_colon {
+            "Use a colon when specifying `noqa` rule codes".to_string()
         } else {
-            format!("Do not add spaces between `noqa` and its colon")
+            "Do not add spaces between `noqa` and its colon".to_string()
         }
     }
```

Comment:
I'd prefer to keep the `let BlanketNoQA { .. } = self;` line for situations where we're unpacking lots of variables at once, like this